editor/animation_track_editor.cpp

```diff
@@ -6153,6 +6175,15 @@ AnimationTrackEditor::AnimationTrackEditor() {
 
 	bottom_hb->add_spacer();
 
+	bezier_edit_icon = memnew(Button);
+	bezier_edit_icon->set_flat(true);
+	bezier_edit_icon->set_disabled(true);
+	bezier_edit_icon->set_toggle_mode(true);
+	bezier_edit_icon->connect("pressed", callable_mp(this, &AnimationTrackEditor::_toggle_bezier_edit));
```

Comment:
```suggestion
	bezier_edit_icon->connect(SNAME("pressed"), callable_mp(this, &AnimationTrackEditor::_toggle_bezier_edit));
```

Reply:
Actually nevermind this one, `SNAME` is not used consistently for signal names in `connect()` calls, that's a matter for another PR (if we confirm that it should be used throughout).